_config.yml

```diff
@@ -26,7 +26,7 @@ defaults:
 
 # Navigation
 navbar-links:
-  Home: ""
+  Home: "https://us-rse.org/"
```

Comment:
Huh, it looks like this isn't being honored. It might be caching - let's give it some time to sit and check on it later.